gemsfx/src/main/java/com/dlsc/gemsfx/templatepane/TemplatePane.java

```diff
@@ -32,12 +32,22 @@ public TemplatePane() {
 
 	private final ObservableMap<Position, Tile> tilesMap = FXCollections.observableHashMap();
 
+        /*
+        AR: ermöglicht das Entfernen von Nodes aus der TemplatePane.
+        */
 	public final Tile setNode(Position pos, Node node) {
 		Tile tile = tilesMap.get(pos);
-		tile.setNode(node);
-		getChildren().add(node);
-		node.setManaged(false);
-		return tile;
+                if (node!=null) {
+            		tile.setNode(node);
+                        getChildren().add(node);
+                        node.setManaged(false);
+                } else {
+                        tile.visibleProperty().unbind();
```

Comment:
The TemplatePane does not bind the "visible" property so there is no need to unbind it. However, it is the TemplatePaneControls class that binds to it, but that is outside the TemplatePane. This code needs to work without the unbinding. It is actually the responsibility of the TemplatePaneControls class to unbind from the visible property when a tile gets removed.

Reply:
I will remove the unbind instructions